osl/interface_linux.go

```diff
@@ -241,8 +241,9 @@ func (n *networkNamespace) AddInterface(srcName, dstPrefix string, options ...If
 	if n.isDefault {
 		i.dstName = i.srcName
 	} else {
-		i.dstName = fmt.Sprintf("%s%d", i.dstName, n.nextIfIndex)
-		n.nextIfIndex++
+		dstNamePrefix := i.dstName
```

Comment:
You are of course correct. Should have read up a few lines in the code. Have made this modification now as well.